src/VisualStudio/Core/Def/Implementation/AnalyzerDependency/AnalyzerDependencyCheckingService.cs

```diff
@@ -80,48 +80,57 @@ internal sealed class AnalyzerDependencyCheckingService
 
             var conflicts = results.Conflicts;
             var missingDependencies = results.MissingDependencies;
+            var projects = _workspace.DeferredState.ProjectTracker.ImmutableProjects;
```

Comment:
the service itself seems designed not to process multiple solution at the same time.... might be better if changed so that service can handle multiple solutions so that things are not blocked.